requirements.txt

```diff
@@ -4,7 +4,7 @@ Flask-Script==2.0.5
 Flask-WTF==0.12
 
 git+https://github.com/madzak/python-json-logger.git@v0.1.3#egg=python-json-logger==v0.1.3
-git+https://github.com/alphagov/digitalmarketplace-utils.git@6.8.1#egg=digitalmarketplace-utils==6.8.1
+git+https://github.com/alphagov/digitalmarketplace-utils.git@6.9.0#egg=digitalmarketplace-utils==6.9.0
```

Comment:
Utils is at 6.9.1 now, although not desperate to bring in the patch change right now particularly.